src/main/java/com/rhombus/api/examples/userList.java

```diff
@@ -0,0 +1,182 @@
+        }
+
+        if (commandLine.hasOption("report"))
+        {
+            reportName = commandLine.getOptionValue("report");
+        }
+        else
+        {
+            reportName = "Report";
+        }
+        if (commandLine.hasOption("names"))
+        {
+            namesList = namesOrganizer(commandLine.getOptionValue("names"));
+        }
+
+        _initialize(apikey);
+        UserGetUsersInOrgWSResponse UserData = getUsersdata();
+        File fileCSV = CSVCreate(reportName, csvName);
+        CSVAdd(UserData, fileCSV, namesList);
+    }
+
+    private static void _initialize(final String apiKey) throws Exception
+    {
+        /*
+         * API CLIENT
+         */
+        {
+            final List<Header> defaultHeaders = new ArrayList<>();
+            defaultHeaders.add(new BasicHeader("x-auth-scheme", "api-token"));
+            defaultHeaders.add(new BasicHeader("x-auth-apikey", apiKey));
+
+            final ClientBuilder clientBuilder = ClientBuilder.newBuilder();
+            clientBuilder.register(new JacksonJaxbJsonProvider().configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false).configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false));;
+
+            final HostnameVerifier hostnameVerifier = new HostnameVerifier() {
+                @Override
+                public boolean verify(String hostname, SSLSession session) {
+                    return true;
+                }
+            };
+
+            clientBuilder.hostnameVerifier(hostnameVerifier);
+
+            _apiClient = new ApiClient();
+            _apiClient.setHttpClient(clientBuilder.build());
+            _apiClient.addDefaultHeader("x-auth-scheme", "api-token");
+            _apiClient.addDefaultHeader("x-auth-apikey", apiKey);
+
+            _userwebservice = new UserWebserviceApi(_apiClient);
+        }
+    }
+
+    public static List<String> namesOrganizer(String namesList)
+    {
+        namesList = namesList.replace(", ",",");
+        List<String> namesArray = Arrays.asList(namesList.split(","));
+        return namesArray;
+    }
+
+    private static UserGetUsersInOrgWSResponse getUsersdata() throws Exception
+    {
+        final UserGetUsersInOrgWSRequest userRequest = new UserGetUsersInOrgWSRequest();
+        final UserGetUsersInOrgWSResponse userResponse = _userwebservice.getUsersInOrg(userRequest);
+        return userResponse;
+    }
+
+    private static File CSVCreate(String reportName, String csvName)
+    {
+        File report = new File(reportName);
+        report.mkdir();
+        File csvFile = new File(report + "/" + csvName + ".csv");
+        if (csvFile.isFile())
+        {
+            System.out.println("File already exists");
+            System.exit(0);
+        }
+        return csvFile;
+    }
+    private static void CSVAdd(UserGetUsersInOrgWSResponse UserData, File csvFile, List <String> namesList) throws Exception
+    {
+        FileWriter csvWriter = new FileWriter(csvFile);
```

Comment:
While this does work, it would be better to use a library for printing csv files.  It's a bit cleaner and easier to parse.  I would use the Apache commons csv printer, linked below.  You will need to add the dependency to the build.gradle file as well.
https://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVPrinter.html